### PR TITLE
Fix resizing issue in Split View on El Capitan

### LINKED ISSUE #93: MacVim resizes itself repeatedly in split view
To reproduce:

1. Create a split view with MacVim and another app.
2. Do anything that causes a screen redraw in MacVim, like editing a file.
3. MacVim pushes the split over to the right.
4. Try to adjust the split to where you want it.
5. As soon as you let go of the drag, MacVim resizes itself and pushes the split to the right.
6. ``:set columns=40`` MacVim sizes to 40 columns...
7. ...but then goes back to taking over most of the screen after certain refresh operations.

My theory is that MacVim is trying to make the view fit the size of the screen; I don't think it understands fully the difference between full screen mode, and split-view mode.

Recording this in case somebody else wants to get to it first.

Before:

screen shot 2015-10-11 at 2 29 11 pm

After:

screen shot 2015-10-11 at 2 30 44 pm

### PR DESCRIPTION
**Pull request by splhack**
Fix #93

### CODE REVIEW

File: src/MacVim/MMWindowController.m [hunk 1]
```diff
@@ -1567,6 +1567,12 @@ - (void)updateToolbar
 
 - (BOOL)maximizeWindow:(int)options
 {
+    if (floor(NSAppKitVersionNumber) > NSAppKitVersionNumber10_10_Max) {
```
Comment (datalogics-kam) [inline src/MacVim/MMWindowController.m]: This basically fixes the problem for me, running on El Capitan. Thanks!

I'm wondering if this conditional should also check to see if the window is actually in split mode.

So far, everything I've tried has worked, even old-style full-screen mode. But I'm wondering if some non-split-screen call to maximizeWindow really does want the code below.

Comment (splhack) [inline src/MacVim/MMWindowController.m]: I had exactly the same thought about that. However I couldn't find any situation that the code is needed on El Capitan. If you find anything, please feel free to create issues.

**Comment by jpetrie** [comment]
:+1: 

This looks good and works for me locally as well.

**Comment by kamitchell** [comment]
(At home now) Built this myself, and it looks good to me.

The other problems with MacVim having redraw problems after resizing the El Capitan full-screen split seem to be unrelated, and for some reason don't always occur. I'll start working with a development version of my own and see if I can puzzle out some of the rare issues.

**Pull request merged**